internal/mithril-aggregator-discovery/src/model.rs

```diff
@@ -0,0 +1,43 @@
+use std::time::Duration;
+
+use mithril_aggregator_client::{AggregatorHttpClient, query::GetAggregatorFeaturesQuery};
+use mithril_common::{StdResult, messages::AggregatorCapabilities};
+
+/// Representation of an aggregator endpoint
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Adding `serde::Serialize` (no need for deserialize) would be interesting to output the result in the cli command when in json.